test/integration/backward_compatible/nearcache/MigratedDataTest.js

```diff
@@ -33,10 +33,9 @@ describe('MigratedDataTest', function () {
 
     async function waitForPartitionTableEvent(partitionService) {
         const deferred = deferredPromise();
-        const expectedPartitionCount = partitionService.partitionCount;
 
         function checkPartitionTable(remainingTries) {
-            if (partitionService.partitionTable.partitions.size === expectedPartitionCount) {
+            if (partitionService.getPartitionOwner(0) != null) {
```

Comment:
Why this change is required? I guess getting the owner of partition 0 do not check if we get all partitions .

Reply:
If you check the mechanism, it is. We set the whole partition table at once which contains the info related to all partitions. I did this change to make use of the public API